cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,151 @@
+from django.test import TestCase
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.standard_pages.tests.factories import IndexPageFactory, InformationPageFactory
+
+
+class IndexPageTestCase(WagtailTestUtils, TestCase):
+    """Test IndexPage model properties and methods."""
+
+    def setUp(self):
+        self.page = IndexPageFactory(
+            title="Test Index Page",
+            description="This is an example",
+            content="This is the main content",
+            featured_pages=None,
+            related_links=None,
+        )
+
+        self.page_url = self.page.url
+
+    def test_no_featured_pages_displayed_when_no_children_and_no_custom_featured_pages_selected(self):
+        """Test that the Featured Items block isn't displayed when the Index Page has no child pages
+        and no custom Featured Items are specified.
+        """
+        self.page.featured_pages = None
+
+        response = self.client.get(self.page_url)
+        self.page.save_revision().publish()
+
+        self.assertEqual(response.status_code, 200)
+
+        self.assertNotContains(response, "ons-document-list")
+
+    def test_children_displayed_as_featured_pages_when_no_custom_featured_pages(self):
+        """Test that the children pages of the Index Page are displayed
+        when no custom Featured Items are specified.
+        """
+        child_page = InformationPageFactory(parent=self.page)
+
+        self.page.save_revision().publish()
```

Comment:
not needed

Reply:
[Removed unnecessary page publication step from the tests](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/ed65f6f3761fba325dd80bb9fcb3f86897331977)

Reply:
[Removed unnecessary page publication step from the tests](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/ed65f6f3761fba325dd80bb9fcb3f86897331977)